src/app/shared/data-table/data-table.component.ts

```diff
@@ -50,4 +50,18 @@ export class DataTableComponent<T> implements OnInit {
 
 
   }
+
+
+  ranges = [
+    {value: 'This Month', viewValue: 'This Month'},
+    {value: 'Previous Mons', viewValue: 'Previous Mons'},
+    {value: '2 Month Ago', viewValue: '2 Month Ago'},
+    {value: '3 Month Ago', viewValue: '3 Month Ago'},
+  ];
+
+  Accounts = [
```

Comment:
Please use camelCase for variables.
Please add public and private to class members and locate them in the top of the class